tests/acceptance/01_vars/02_functions/network/url_get.cf

```diff
@@ -27,8 +37,19 @@ bundle agent init
       "options_clean" data => '{}';
       "res1" data => url_get("http://acme.com", options_clean);
       "res2" data => url_get("http://nosuchcfenginehost.com", options_clean);
-      # static content, we hope won't change much
-      "res3" data => url_get("http://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js", options_limit);
+      "res3" data => url_get("http://localhost:8000/index.html", options_limit);
+
+  commands:
+    "/usr/bin/python3 -m http.server 8000"
```

Comment:
You should be able to omit `/usr/bin/` since you're using useshell.

Reply:
:+1: